variant-remapping-automation/prepare_remapping_table.py

```diff
@@ -0,0 +1,246 @@
+                         release_version, num_studies, num_ss_ids, study_accessions):
+    with get_metadata_connection_handle('production_processing', private_config_xml_file) as pg_conn:
+        query = f"insert into eva_progress_tracker.remapping_tracker (source, taxonomy, scientific_name, " \
+                f"origin_assembly_accession, assembly_accession, remapping_version, release_version, " \
+                f"num_studies, num_ss_ids, study_accessions) " \
+                f"values ('source_placeholder',{tax},'{scientific_name}','{org_asm}','{target_asm}'," \
+                f"'{remapping_version}', {release_version}, " \
+                f"{num_studies}, {num_ss_ids}, {study_accessions}) "
+        if 'EVA' in sources:
+            query_eva = query.replace('source_placeholder', 'EVA')
+            logger.info(f"{query_eva}")
+            execute_query(pg_conn, query_eva)
+        if 'DBSNP' in sources:
+            query_dbsnp = query.replace('source_placeholder', 'DBSNP')
+            logger.info(f"{query_dbsnp}")
+            execute_query(pg_conn, query_dbsnp)
+
+
+def get_tax_all_supported_assembly_from_eva(private_config_xml_file):
+    with get_metadata_connection_handle('production_processing', private_config_xml_file) as pg_conn:
+        query = f"""select taxonomy_id, assembly_id from evapro.supported_assembly_tracker"""
+
+        tax_asm_list = {}
+        for tax, asm in get_all_results_for_query(pg_conn, query):
+            if tax not in tax_asm_list:
+                tax_asm_list[tax] = {asm}
+            else:
+                tax_asm_list[tax].add(asm)
+
+        return tax_asm_list
+
+
+def get_assembly_latest_remapping_time(private_config_xml_file):
+    asm_remaptime = {}
+    # TODO: change to production_processing after filling data
+    with get_pg_accession_connection_handle('development', private_config_xml_file) as pg_conn:
+        query = f"""select bjep.string_val, max(bje.start_time)
+                    from batch_job_execution_params bjep 
+                    join batch_job_execution bje on bjep.job_execution_id = bje.job_execution_id 
+                    join batch_job_instance bji on bji.job_instance_id = bje.job_instance_id 
+                    where bjep.key_name = 'assemblyAccession'
+                    and bji.job_name = 'INGEST_REMAPPED_VARIANTS_FROM_VCF_JOB'
+                    and bje.status = 'COMPLETED' and bje.exit_code = 'COMPLETED'
+                    group by bjep.string_val 
+                    order by bjep.string_val"""
+
+        for assembly, remap_time in get_all_results_for_query(pg_conn, query):
+            asm_remaptime[assembly] = remap_time
+        return asm_remaptime
+
+
+def get_studies_for_remapping(private_config_xml_file):
+    asm_remaptime = get_assembly_latest_remapping_time(private_config_xml_file)
+    # TODO: change to production_processing after filling data
+    with get_pg_accession_connection_handle('development', private_config_xml_file) as pg_conn:
```

Comment:
If I understand correctly, accessioning has always been done correctly in `production_processing`, it's just the remapping ingestion that erroneously occurred in `development`...

Reply:
Yes,  you are correct. updated to production_processing